src/core/ledger/diffLedger.test.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+
+import {diffLedger} from "./diffLedger";
+import * as uuid from "../../util/uuid";
+import {Ledger} from "./ledger"; // for spy purposes
+
+describe("core/ledger/diffLedger", () => {
+  const randomMock = jest.spyOn(uuid, "random");
+
+  let nextFakeUuidIndex = 0;
+  function resetFakeUuid() {
+    nextFakeUuidIndex = 0;
+  }
+  function nextFakeUuid(): uuid.Uuid {
+    const uuidString = String(nextFakeUuidIndex).padStart(21, "0") + "A";
+    nextFakeUuidIndex++;
+    return uuid.fromString(uuidString);
+  }
+
+  randomMock.mockImplementation(nextFakeUuid);
+  function setNextUuid(x: uuid.Uuid) {
+    randomMock.mockImplementationOnce(() => x);
+  }
```

Comment:
I believe this is copy-pasta'd from ledger. Would be great to have a separate PR which just creates a `fakeUuid` module (or `uuidTestUtil`) which exports this functionality.

Reply:
Done in this PR: https://github.com/sourcecred/sourcecred/pull/2505